templates/enduro.toml.j2

```diff
@@ -147,26 +147,29 @@ bucket = "aips"
 [failedPips]
 endpoint = "{{ pipeline.watched_endpoint }}"
 pathStyle = true
-key = "{{ pipeline.watched_key }}"
-secret = "{{ pipeline.watched_secret }}"
+accessKey = "{{ pipeline.watched_key }}"
+secretKey = "{{ pipeline.watched_secret }}"
 region = "{{ pipeline.watched_region }}"
 bucket = "failed-pips"
```

Comment:
failed-pips and failed-sips should be added to the data dictionary, as we do with the sips one

Reply:
Not really related with these changes and I don't want to introduce two new variables in the playbooks. I also think this block shouldn't be part of the `enduro_sfa_pipelines` loop and other improvements that we already discussed.

Better if we do those changes in other PRs after you are done with the splitting of roles you mentioned in Slack.